src/constants.ts

```diff
@@ -1,2 +1,8 @@
 export const LIST_SECRETS_MAX_RESULTS = 100;
-export const CLEANUP_NAME = 'SECRETS_LIST_CLEAN_UP';
\ No newline at end of file
+export const CLEANUP_NAME = 'SECRETS_LIST_CLEAN_UP';
+
+export enum NameTransformation {
```

Comment:
Maybe instead of enum to avoid switch case in `transformToValidEnvName` function. Something like this could be good for posterity: 

```
export const NAME_TRANSFORMATION = { 
    "None": x => x, 
    "Uppercase": x => x.toUpperCase(), 
    "Lowercase": x => x.toLowerCase() 
}
```

Then in the `transformToValidEnvName` you just do something like:
```
const transformName = NAME_TRANSFORMATION[nameTransformation];
const transformedName = transformName(secretName);
if (!transformedName) {
   throw new Error(`'${nameTransformation}' is unsupported name transformation.`);
}
return transformedName;
```

Reply:
you would also have to change it to loop over `Object.keys` in `parseNameTransformation`

Reply:
Thanks,
I modified it a bit, and took the idea with having just the transformation function passed around without explicitly specifying implementation inside of the  `transformToValidEnvName`.